reana_client/cli/files.py

```diff
@@ -40,6 +40,7 @@ def files_group(ctx):
 
 @files_group.command('ls')
 @add_workflow_option
+@check_connection
```

Comment:
Can we pass the `@check_connection` decorator to the parent Click group `files_group `? All file commands need the check so we write it fewer times.

Same happens to `@add_access_token_options` which we are writing several times. Since this one is not related to the PR you can leave, but if it is quick for you to test it would be better to move it together with `@check_connection` to the Click group.

Reply:
> Can we pass the @check_connection decorator to the parent Click group files_group ? All file commands need the check so we write it fewer times.

It was the initial idea, unfortunately it does not work.